test/annotation_utils.t

```diff
@@ -74,7 +76,18 @@
 my $obj_65386_1 = '65386/2/1';  # same as 63171/436/1, i.e., GCF_003058445.1
 my $obj_65386_2 = '65386/12/1';
 
-my $asmb_fasta = $annoutil->_get_fasta_from_assembly($obj_asmb);
+my $asmb_fasta;
+
+lives_ok {
+    $asmb_fasta = $annoutil->_get_fasta_from_assembly($obj_asmb);
+} 'Got FASTA from saved assembly file';
+
+unless ( $asmb_fasta ) {
+    fail 'Could not set up test data; skipping all tests';
+    RASTTestUtils::clean_up();
+    done_testing;
+    exit 1;
+}
```

Comment:
I don't have access to 55141, so this catches the error a little more nicely and skips the rest of the tests.